diff --git a/node_modules/@astrojs/internal-helpers/dist/fs.js b/node_modules/@astrojs/internal-helpers/dist/fs.js
index 8e78d21..541ae99 100644
--- a/node_modules/@astrojs/internal-helpers/dist/fs.js
+++ b/node_modules/@astrojs/internal-helpers/dist/fs.js
@@ -36,6 +36,8 @@ async function copyFilesToFolder(files, outDir, exclude = []) {
     }
   }
   for (const origin of fileList) {
+    if (origin.includes("es5-ext")) continue;
+    if (origin.includes("es5-ext")) continue;
     const dest = new URL(nodePath.relative(commonAncestor, origin), outDir);
     const realpath = await fs.realpath(origin);
     const isSymlink = realpath !== origin;
@@ -49,7 +51,7 @@ async function copyFilesToFolder(files, outDir, exclude = []) {
       const realdest = fileURLToPath(new URL(nodePath.relative(commonAncestor, realpath), outDir));
       const target = nodePath.relative(fileURLToPath(new URL(".", dest)), realdest);
       if (!existsSync(dest)) {
-        await fs.symlink(target, dest, isDir ? "dir" : "file");
+        try { await fs.symlink(target, dest, isDir ? "dir" : "file"); } catch (err) { if (err.code !== "EEXIST") throw err; }
       }
     } else if (!isDir) {
       await fs.copyFile(origin, dest);
